### PR TITLE
pt_BR translation

### PR DESCRIPTION
**Pull request by DenysLins**
I added some translations to pt_BR

### CODE REVIEW

**Comment by DenysLins** [comment]
Verdade... tinha a impressão que havia mudado com a reforma ortográfica. Mudei onde deveria mudar.

File: core/src/main/resources/hudson/Messages_pt_BR.properties [hunk 1]
```diff
@@ -76,3 +76,11 @@ ProxyConfiguration.FailedToConnect=Falha ao conectar em {0} (c\u00f3digo {1}).
 ProxyConfiguration.Success=Sucesso
 # Did not manage to validate {0} (may be too slow)
 FilePath.did_not_manage_to_validate_may_be_too_sl=N\u00e3o conseguiu validar {0} (pode ser muito lento)
+# \u2018{0}\u2019 doesn\u2019t match anything because it is treated case sensitively. You can deactivate case sensitivity to get matches
```
Comment (daniel-beck) [inline core/src/main/resources/hudson/Messages_pt_BR.properties]: This file is missing the actual localization.

**Comment by daniel-beck** [comment]
Is this a work on progress? Several files are missing the translated texts.

**Comment by DenysLins** [comment]
Yes. It's an incremental job...

**Comment by daniel-beck** [comment]
@DenysLins What I mean is that if this were merged as is, a lot of labels would not be English, but contain no text at all.

**Comment by oleg-nenashev** [comment]
The PR is not mergeable anymore. Needs a rebase

**Comment by frekele** [comment]
I translated all fields today, in the translation plugin with the Jenkins 2.0-beta2 version. Check please.

File: core/src/main/resources/jenkins/widgets/HistoryPageFilter/queue-items_pt_BR.properties [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+
+cancel\ this\ build=cancelar esta build
+Expected\ build\ number=
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/widgets/HistoryPageFilter/queue-items_pt_BR.properties]: No translation.

**Comment by daniel-beck** [comment]
@frekele We're currently working hard at getting 2.0 out the door, but I took a note to do that. Unfortunately, that's a bit more involved.

**Comment by oleg-nenashev** [comment]
Merge conflict :(

**Comment by oleg-nenashev** [comment]
@DenysLins Ping. Any plans to finalize it?

**Comment by oleg-nenashev** [comment]
@DenysLins @boaglio @frekele 
Is there something left in this PR to be merged since last translation updates? If no, I propose to close it

**Comment by oleg-nenashev** [comment]
Closing since there is no response from @DenysLins after several pings

**Pull request closed without merging**